### DISCUSSION TITLE
点歌：强化学习和深度强化学习重要论文串烧

### DISCUSSION THREAD

**Discussion by green-dalii** [mli/paper-reading#128] [General]
待补充：
https://arxiv.org/abs/1509.06461

**Comment by dourgey** [comment]
补充几个个人认为在DRL历史上重要的几篇paper：
* 经典的 [NatureDQN](https://www.nature.com/articles/nature14236)
* DQN系列集大成之作 [Rainbow](https://arxiv.org/abs/1710.02298)
* 几乎成为DRL落地首选的 [PPO](https://arxiv.org/abs/1707.06347)
* Actor-Critic框架下的经典作品 [A3C](https://arxiv.org/abs/1602.01783)

**Reply by hijkzzz** [reply]
外加 AlphaGo AlphaStar

**Comment by shan23chen** [comment]
外加PPO，和最近小火的InstructGPT

**Comment by qzweng** [comment]
鉴于沐神的系统背景，也许对 RL + System 的工作也会有讨论的兴趣吧？供参考：
- DeepMind: ICML 2018 [IMPALA](http://proceedings.mlr.press/v80/espeholt18a.html)
- Google: ICLR 2020 [Seed RL](https://openreview.net/forum?id=rkgvXlrKwH)
